qiskit/transpiler/target.py

```diff
@@ -1306,6 +1313,7 @@ def from_configuration(
             pulse_alignment=pulse_alignment,
             acquire_alignment=acquire_alignment,
             qubit_properties=qubit_properties,
+            meas_map=format_meas_map(meas_map) if isinstance(meas_map, list) else meas_map,
```

Comment:
And again, the inconsistency between the doc string and the actual format used later.